JuliaInterface/src/JuliaInterface.c

```diff
@@ -207,6 +207,107 @@ Obj NewJuliaFunc(jl_function_t* function, int autoConvert)
     return func;
 }
 
+/*
+ * C function pointer calls
+ */
+
+typedef Obj (*func_ptr_0_args)();
+Obj DoCallJuliaCFunc0Arg( Obj func )
+{
+    func_ptr_0_args function = (func_ptr_0_args)jl_unbox_voidpointer((jl_value_t*)FEXS_FUNC(func));
```

Comment:
I would put `jl_unbox_voidpointer((jl_value_t*)FEXS_FUNC(func))` into a helper function. That'll make the code more readable, and also make it trivial to change how the func pointer is stored.